x-pack/plugin/esql/qa/testFixtures/src/main/resources/unmapped_fields.csv-spec

```diff
@@ -495,8 +495,8 @@ required_capability: source_field_mapping
 required_capability: unmapped_fields
 FROM partial_mapping_sample_data,partial_mapping_excluded_source_sample_data METADATA _index
 | INSIST_🐔 message
-| SORT message, @timestamp
 | STATS max(@timestamp), count(*) BY message
+| SORT message NULLS FIRST
```

Comment:
Why this change?

Reply:
The ordinal grouping operator emits output sorted by the keys, while the hash operator outputs keys in arrival order.